### PR TITLE
Setup JsDoc for API autogeneration

### PR DESCRIPTION
**Pull request by risenW**
This pull fixes issue #97

### CODE REVIEW

**Review by mariorodeghiero — APPROVED** [review]
looks good

**Comment by risenW** [comment]
@kmanaseryan What do you think of this pull? ready to merge?

File: README.md [hunk 1]
```diff
@@ -160,264 +160,10 @@ Dataset.load(path).then(dataset => {
...
-@returns Promise with readable stream object on resolve
-
-#### buffer
-
-`File.buffer`
-
-Get this file as a buffer (async)
-
-@returns: promise which resolves to the buffer
+See API documentation [here](./docs)
```
Comment (kmanaseryan) [inline README.md:152]: A lot of headings have been removed from API section, so the table of content above needs to be updated.

**Review by kmanaseryan — CHANGES_REQUESTED** [review]
@risenW looks nice, just requested a one small update for table of contents. I guess you can also resolve the conflicts when you do this change.

File: README.md (hunk 1 above)
Comment (risenW) [reply README.md:152]: Okay, thanks for noting this.

**Review by steveoni — APPROVED** [review]

**Pull request merged**